apps/api/src/main.ts

```diff
@@ -13,7 +13,7 @@ async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   const globalPrefix = 'api';
   app.setGlobalPrefix(globalPrefix);
-  let origin = process.env.CTIMS_ENV === 'development' ? '*' : 'https://ctims.ca';
+  let origin = process.env.CTIMS_ENV === 'development' ? '*' : 'http://localhost:3000';
```

Comment:
Do not modify.

Reply:
If I leave it in its original form, then the CTIMS home page does not load properly. I'll revert to original form before making the final commit

Reply:
CITMS_ENV should come from the .env file. First copy the .env.example file with the CITMS_ENV variable in it to same directory and name it .env. Next set the value to "development" then it should make origin "*" which should make everything work.

Reply:
Got it. Ill do that now